requirements/quality.in

```diff
@@ -3,7 +3,7 @@
 -r test.txt               # Core and testing dependencies for this package
 
 caniusepython3            # Additional Python 3 compatibility pylint checks
-edx-lint                  # edX pylint rules and plugins
+edx-lint==1.3.0           # There is a compatibility issues with newer versions of pylint-django and wagtail that will be fixed in https://github.com/wagtail/wagtail/pull/5538
```

Comment:
Not an ideal fix, but seems better than pinning things in edx-lint to account for the incompatability issues. We should be able to unpin when we upgrade to >wagtail 2.7